x-pack/plugins/security_solution/public/timelines/store/timeline/helpers.ts

```diff
@@ -156,6 +156,7 @@ export const addTimelineToStore = ({
     ...timelineById,
     [id]: {
       ...timeline,
+      filterManager: timelineById[id].filterManager,
```

Comment:
action ADD_TIMELINE wiped out the filterManager we set when initialising t-grid

Reply:
Awesome, thank you!

Reply:
filterManager shouldn't ever be stored in state?